test/controllers/UploadControllerSpec.scala

```diff
@@ -218,6 +218,54 @@ class UploadControllerSpec extends FrontEndTestHelper {
         status(uploadPage) mustBe FORBIDDEN
       }
     }
```

Comment:
Is it worth adding a line to the `render the judgment upload is complete page if the upload has completed` test and the `render the upload is complete page if the upload has completed` to make sure the correct link is shown?

Reply:
The "Upload has completed" page and the "Upload in progress page", you mean? Do you mean like checking for text that is specific to these pages? Because I was thinking this but then realised that if you access the judgment page when you are a standard user (and vice versa), it just shows the "You are not permitted to see this page This page is restricted to certain users only." message and nothing "progress"/"completed" page-specific.

Reply:
For those two tests I mentioned, they're checking for the `Your upload is complete and has been saved` text. You've made changes now so the button link is different depending on the user so I was wondering whether it was worth adding a check at the end of both of those tests for the link address.

Reply:
But the link address is the same for all 3, they all end in `/upload`. Should we maybe change that and have `/upload-has-completed` and `/upload-in-progress`? On the negative side, the users might save that link and try to access the platform via it, on the positive side, we can use it for the tests.

Reply:
I meant the link in the button, the one that has an if condition here https://github.com/nationalarchives/tdr-transfer-frontend/blob/8a33ae06d37ae358dc5eae7d59d3cfb8328893fb/app/views/uploadHasCompleted.scala.html#L12
The difference is only /consignment/{id}/records vs /judgment/{id}/records so it's probably not worth testing for.